extensions/resteasy-server-common/spi/src/main/java/io/quarkus/resteasy/server/common/spi/FriendlyJaxRsAnnotationPrefixBuildItem.java

```diff
@@ -0,0 +1,18 @@
+package io.quarkus.resteasy.server.common.spi;
+
+import org.jboss.jandex.DotName;
+
+import io.quarkus.builder.item.MultiBuildItem;
+
+public final class FriendlyJaxRsAnnotationPrefixBuildItem extends MultiBuildItem {
+
+    private final DotName annotationClass;
```

Comment:
We usually use plain strings for the class names in the build items.

Reply:
OK, I used plain strings